flink-tests/src/test/java/org/apache/flink/test/checkpointing/EventTimeWindowCheckpointingITCase.java

```diff
@@ -162,8 +163,7 @@ private Configuration getConfiguration() throws Exception {
         // Testing HA Scenario / ZKCompletedCheckpointStore with incremental checkpoints
         StateBackendEnum stateBackendEnum = getStateBackend();
         if (ROCKSDB_INCREMENTAL_ZK.equals(stateBackendEnum)) {
-            zkServer = new TestingServer();
-            zkServer.start();
+            zkServer = ZooKeeperTestUtils.createZookeeperTestingServer();
```

Comment:
Should we switch to `zkServer.close()` here as well to be consistent with the other test cases? I guess, `close` is appropriate here because we removing the reference here as well. On close also deletes the data directory...